JSX within objects not parsed

First, thank you for this excellent plugin. I just wanted to raise a small issue - lately I've been using Storybook at work, and I noticed that the plugin does not seem to parse classNames when the JSX is contained within an object. This is pretty typical for Storybook when using the `render` method, e.g.

```tsx
const Example: Story = {
  render: () => (
    <div className="rounded-md h-48 w-48 relative after:h-8 after:w-8 after:absolute after:right-4 after:rotate-45">
      Some content
    </div>
  )
}
```

FWIW [eslint-plugin-tailwindcss](https://github.com/francoismassart/eslint-plugin-tailwindcss) does not have this issue and is still able to lint this JSX.

In our case ESLint is working as it should on .tsx files. If I run ESLint on the file in question, no errors are raised. I've also confirmed that `readable-tailwind` rules are being applied by running `npx eslint --print-config path/to/file.tsx`.

If I extract the JSX into a function like a normal React component, the classNames are properly flagged with the readable-tailwind/multiline rule:

```tsx
// Not flagged by ESLint
export const ExampleStory: Story = {
  render: () => (
    <div className="tw-rounded-none tw-h-48 tw-w-48 tw-relative after:tw-h-8 after:tw-w-8 after:tw-absolute after:-tw-right-4 after:tw-rotate-45">
      Some content
    </div>
  ),
};

// Flagged by ESLint
export const ExampleComponent = () => (
  <div className="tw-rounded-none tw-h-48 tw-w-48 tw-relative after:tw-h-8 after:tw-w-8 after:tw-absolute after:-tw-right-4 after:tw-rotate-45">
    Some content
  </div>
);
```

With the above, when I run eslint directly on the file, the `ExampleComponent` is correctly flagged by the `readable-tailwind/multiline` rule with this output from ESLint:

```
/path/to/file.stories.tsx
  135:18  error  Incorrect line wrapping. Expected

"tw-relative·tw-h-48·tw-w-48·tw-rounded-none·after:tw-absolute·after:-tw-right-4·after:tw-h-8·after:tw-w-8·after:tw-rotate-45"

to be

`↵
····tw-relative·tw-h-48·tw-w-48·tw-rounded-none↵
····after:tw-absolute·after:-tw-right-4·after:tw-h-8·after:tw-w-8·after:tw-rotate-45↵
··`  readable-tailwind/multiline
```

Hope this makes sense! Let me know if there are any more details I can provide.

--- a/src/options/default-options.ts
+++ b/src/options/default-options.ts
@@ -34,6 +34,7 @@ export const DEFAULT_CALLEE_NAMES = [
 
 export const DEFAULT_ATTRIBUTE_NAMES = [
   "class",
+  "className",
   [
     "class", [
       {

